lib/api/apiUtils/object/BackendInfo.js

```diff
@@ -140,6 +145,12 @@ class BackendInfo {
             `Your bucket location "${escapeForXml(bucketLocationConstraint)}"` +
             ' is not in your location config - Please update.' };
         }
+        const legacyLocationConstraint =
+        BackendInfo.getLegacyLocationConstraint();
+        if (legacyLocationConstraint) {
+            log.trace('legacy location is valid');
+            return { isValid: true, legacyLocationConstraint };
```

Comment:
do you need the `legacyLocationConstraint` property returned for anything else than feeding it to the constructor to instantiate the class?

Reply:
not for now.